lib/image_optim/cmd.rb

```diff
@@ -44,6 +86,38 @@ def check_status!
 
         fail SignalException, status.termsig
       end
+
+      def init_options!(args)
+        pgroup_opt = Gem.win_platform? ? :new_pgroup : :pgroup
+
+        if args.last.is_a?(Hash)
+          args.last[pgroup_opt] = true
+        else
+          args.push(pgroup_opt => true)
+        end
+      end
+
+      def cleanup_process(pid)
+        Thread.new do
+          Process.kill('-TERM', pid)
```

Comment:
From `Process.kill`:

> If signal is negative (or starts with a minus sign), kills process groups instead of processes.

I'm pretty sure that isn't what you want to have happen here.  A straight-up `TERM` might be more appropriate.

(Also applies to the `-KILL` on line 117)